### PR TITLE
(GH-972) Implement Chat Feature

### PR DESCRIPTION
**Pull request by designxsteph**
Implements a chat feature that users can ask/interact with Chocolatey
members about Chocolatey for Business. This is restricted to certain
pages that users might navigate to to find out more about the Business
features Chocolatey offers.

### CODE REVIEW

**Review by ferventcoder — CHANGES_REQUESTED** [review]
See if we can move the main bulk of the function out to a js file, then load that in on bundles.reference. That way we can reduce the amount of duplication across multiple pages.

We can pass the chatbotid into the function once loaded and let it decide if it is null or not, and decide to load up. 

See if we can load it at the bottom as well, it doesn't seem like it needs the priority to load with top scripts.

**Comment by designxsteph** [comment]
@ferventcoder This has been fixed and new version is up on test now.

File: chocolatey/Website/Scripts/chat/chat.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+function chatBot(chatBotId) {
+    "use strict";
+
+    !function () {
```
Comment (gep13) [inline chocolatey/Website/Scripts/chat/chat.js:4]: @designxsteph should there still be a check in here to verify that the chatBotId isn't null, before attempting to use it?

**Review by gep13 — CHANGES_REQUESTED** [review]

File: chocolatey/Website/Scripts/chat/chat.js (hunk 1 above)
Comment (designxsteph) [reply chocolatey/Website/Scripts/chat/chat.js]: @gep13 Ah yeah, I added that in!

**Comment by designxsteph** [comment]
@ferventcoder this should be on all the pages we want it on now, let me know if there are more. A new version of this is on test.

**Review by ferventcoder — APPROVED** [review]

**Pull request merged**